pulpcore/app/importexport.py

```diff
@@ -0,0 +1,80 @@
+import os
+
+from import_export import resources
+from pulpcore.app.models.content import (
+    Artifact,
+    Content,
+    ContentArtifact,
+)
+from pulpcore.app.models.repository import (
+    Repository,
+    RepositoryVersion,
+)
+
+
+class RepositoryResource(resources.ModelResource):
```

Comment:
These seem to duplicate what's in `modelresource.py`?

Reply:
Indeed - leftover froma  refactoring